samples/client/petstore/rust/src/apis/mod.rs

```diff
@@ -0,0 +1,31 @@
+use hyper;
+use serde_json;
+
+#[derive(Debug)]
+pub enum Error {
+    Hyper(hyper::Error),
+    Serde(serde_json::Error),
+}
+
+impl From<hyper::Error> for Error {
+    fn from(e: hyper::Error) -> Self {
+        return Error::Hyper(e)
+    }
+}
+
+impl From<serde_json::Error> for Error {
+    fn from(e: serde_json::Error) -> Self {
+        return Error::Serde(e)
+    }
+}
+
+use super::models;
+
+mod get_pet_by_id_api;
```

Comment:
there's a single namespace, so modules (and files) would have to be named differently from the actual functions.